src/emitter/emitter.ts

```diff
@@ -35,6 +49,39 @@ export class EventEmitter<E extends object> implements IEventEmitter<E> {
   public dispose() {
     this._emitter.removeAllListeners();
   }
+
+  public createEmitGroup(): {
+    queueCount: number;
+    emit: EventEmitter<E>['emit'];
+    flush: () => void;
+    reset: () => void;
+  } {
+    const notifier = this;
+
+    let eventQueue: Array<[unknown, Array<unknown>]> = [];
```

Comment:
```suggestion
    const eventQueue: Array<[string | number, Array<unknown>]> = [];
```
In case the typings were specified.